egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -493,6 +493,11 @@ def TrainOneIteration(dir, iter, egs_dir,
     elif os.stat(new_model).st_size == 0:
         raise Exception("{0} has size 0. Something went wrong in iteration {1}".format(new_model, iter))
 
+    try:
+        os.remove("{dir}/cache.{iter}".format(dir = dir, iter = iter))
+    except OSError:
+        pass
```

Comment:
nitpick: IMO silencing exceptions like it's done here, goes against their intended use(reporting unexpected events). Personally I tend to prefer something like:
```
if os.path.exists(<file>):
    os.remove(<file>)